src/EventStore.Client/EventStoreClientSettings.ConnectionString.cs

```diff
@@ -191,23 +182,39 @@ private static class ConnectionStringParser {
 					connSettings.GossipOverHttps = useTls;
 				}
 
+#if !GRPC_CORE
+				settings.CreateHttpMessageHandler = () => {
+					var handler = new SocketsHttpHandler();
+
+					if (typedOptions.TryGetValue(TlsVerifyCert, out var tlsVerifyCert) && !(bool)tlsVerifyCert) {
+						handler.SslOptions.RemoteCertificateValidationCallback = delegate { return true; };
+					}
+
+					if (settings.ConnectivitySettings.KeepAlive.HasValue) {
+						handler.KeepAlivePingDelay = settings.ConnectivitySettings.KeepAlive.Value;
+					}
+
+					return handler;
+				};
+#endif
+
```

Comment:
@thefringeninja if `tlsVerifyCert` is `false` and we're using GRPC_CORE, should we throw an exception? Based on gRPC's source it does not appear to expose an option to disable server certificate validation (there's an option called `GRPC_TLS_SKIP_ALL_SERVER_VERIFICATION` in the TLS security connector library but i think that the Grpc.Core c# library does not use it (https://github.com/grpc/grpc/blob/e745f8cb558798f1c8b91baa42d46228a9d4a0c4/src/core/lib/security/security_connector/tls/tls_security_connector.cc#L380)

Reply:
It works a bit differently in gRPC core... if you don't send `ChannelCredentials` then it is ok to communicate over HTTP. What we could do here is thrown exception or write warning in the logs if they set channel credentials and set `tls=false`.